src/4.3_to_5.0/__test_resources__/smallLegacyUIFolderWithInvalidKpiTitle.ts

```diff
@@ -0,0 +1,241 @@
+/**
+ * The shortened version of the content of the /ui folder on a Content Server, useful for unit tests.
+ * Contains a KPI widget containing a custom title with an empty `tupleKey` for its location.
+ */
+export const smallLegacyUIFolderWithInvalidKpiTitle = {
```

Comment:
It would be nice to factorize the common part between this and the test resource introduced in #128, in order to avoid adding redundant content to this repo.

For instance, we could have a `emptyLegacyUIFolder` test resource and a `addLegacyBookmarkToUIFolder` test resource function that takes a map of id to legacy bookmark as argument and returns the "small legacy UI folder with ..." object, without mutating `emptyLegacyFolder`.